### ISSUE TITLE
Slashes in URL fragments get URL-encoded when the link is clicked on in *.md preview

### ISSUE THREAD

**Issue by orucky** [theia-ide/theia#2435]
Assume the *.md file contains this link:

  https://gitpod.io#https://github.com/arunoda/learnnextjs-demo

In the preview this is rendered as a clickable link.

1. if clicked on using the left button, a browser tab with the URL with the slashes in the fragment URL-encoded is started (in this case: https://gitpod.io/#https%3A%2F%2Fgithub.com%2Farunoda%2Flearnnextjs-demo), which is wrong (different URL)

2. however, if a right button click -> Open in a new XYZ is used, all works fine.

**Comment by marechal-p** [comment]
After a bit more thoughts, this is even more dangerous than I thought: This will load any remote content in the electron window, when we currently allow every NodeJS API.
@akosyakov @svenefftinge

**Comment by svenefftinge** [comment]
@marechal-p true. If the link fetches js code that uses node API it could do quite some interesting things on your machine.
